From dc3729f72ca764439ac609a7a031943afa75fb8a Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sat, 22 Mar 2025 10:20:07 -0400
Subject: [PATCH 9/9] Skip problematic tests for linux_ppc64le_mpimvapich

	109 - MPI_TEST_testphdf5 (Failed)
	118 - MPI_TEST_testphdf5_cschunkw (Failed)
	119 - MPI_TEST_testphdf5_ccchunkw (Failed)
	120 - MPI_TEST_testphdf5_tldsc (Failed)
	127 - MPI_TEST_t_pflush1 (Timeout)
	128 - MPI_TEST_t_pflush2 (Failed)
	133 - MPI_TEST_t_select_io_dset (Timeout)
	136 - MPI_TEST_t_2Gio (Failed)
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 93d9e4f..9df9836 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -256,9 +256,9 @@ set (HDF_CONFIG_DIR            ${HDF5_SOURCE_DIR}/config)
 set (HDF_RESOURCES_DIR         ${HDF5_SOURCE_DIR}/config/cmake)
 set (HDF5_SRC_DIR              ${HDF5_SOURCE_DIR}/src)
 set (HDF5_TEST_SRC_DIR         ${HDF5_SOURCE_DIR}/test)
-set (HDF5_TEST_PAR_DIR         ${HDF5_SOURCE_DIR}/testpar)
+# set (HDF5_TEST_PAR_DIR         ${HDF5_SOURCE_DIR}/testpar)
 set (HDF5_TEST_API_SRC_DIR     ${HDF5_SOURCE_DIR}/test/API)
-set (HDF5_TEST_API_PAR_SRC_DIR ${HDF5_SOURCE_DIR}/testpar/API)
+# set (HDF5_TEST_API_PAR_SRC_DIR ${HDF5_SOURCE_DIR}/testpar/API)
 set (HDF5_CPP_SRC_DIR          ${HDF5_SOURCE_DIR}/c++/src)
 set (HDF5_CPP_TST_DIR          ${HDF5_SOURCE_DIR}/c++/test)
 set (HDF5_HL_SRC_DIR           ${HDF5_SOURCE_DIR}/hl/src)
-- 
2.34.1

